src/ResourceManager/Network/Commands.Network/help/Add-AzureRmExpressRouteCrossConnectionPeering.md

```diff
@@ -0,0 +1,320 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+ms.assetid: C44AD23A-E575-418C-BE90-323B44D6D2E8
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/add-azurermexpressroutecrossconnectionpeering
+schema: 2.0.0
+---
+
+# Add-AzureRmExpressRouteCrossConnectionPeering
+
+## SYNOPSIS
+Adds a peering configuration to an ExpressRoute cross connection.
+
+## SYNTAX
+
+### SetByResource (Default)
+```
+Add-AzureRmExpressRouteCrossConnectionPeering -Name <String> -ExpressRouteCrossConnection <PSExpressRouteCrossConnection>
+ -PeeringType <String> -PeerASN <UInt32> -PrimaryPeerAddressPrefix <String> -SecondaryPeerAddressPrefix <String>
+ -VlanId <Int32> [-SharedKey <String>]
+ [-MicrosoftConfigAdvertisedPublicPrefixes <System.Collections.Generic.List`1[System.String]>]
+ [-MicrosoftConfigCustomerAsn <Int32>] [-MicrosoftConfigRoutingRegistryName <String>]
+ [-PeerAddressType <String>] [-LegacyMode <Boolean>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### MicrosoftPeeringConfigRoutFilterId
```

Comment:
Where are these 2 other parameter sets coming from? They are the same as the default.

Reply:
Fixed